tools/dhtnode.cpp

```diff
@@ -101,13 +113,17 @@ void cmd_loop(std::shared_ptr<DhtRunner>& dht, dht_params& params)
 #if OPENDHT_PROXY_SERVER
```

Comment:
This up to line 128 should not be part of the cmd_loop, should go to main, to be respected even in non-interactive modes. Maybe make proxies a global, of pass reference to cmd_loop ?

Reply:
I prefer the reference.